heaps/min_heap.py

```diff
@@ -15,24 +15,36 @@ class MinHeap:
     def __init__(self):
         self.store = []
 
-
     def add(self, key, value = None):
```

Comment:
👀 Nice! Space complexity would be O(log n) because of the recursive call stack of `heap_up` however!